docs/pre-release/reference/config.md

```diff
@@ -46,6 +58,16 @@ These are the valid fields for the `logLevels` key:
 |`userDaemon`|Logging level to be used by the User Daemon (logs to connector.log)|debug|
 |`rootDaemon`|Logging level to be used for the Root Daemon (logs to daemon.log)|info|
 
+## Environment Variables
```

Comment:
Will remove this chunk bc we want to reduce the use of environment variables (maybe even remove them completely?)